### ISSUE TITLE
No connection when expanding in RTX, but do have connections in KG2c

### ISSUE THREAD

**Issue by dkoslicki** [RTXteam/RTX#1832]
Going through ARAX, Valproic acid (from the question of the month) appears to have no connections: https://arax.ncats.io/?r=40836
Yet when going to http://kg2-7-5c.rtx.ai:7474/browser/ and running `match p=(n{id:"CHEMBL.COMPOUND:CHEMBL109"})-[]-(m) return p limit 10` I do get connections.

So I'm thinking this might be an `expand` issue rather than a KG2c issue.

**Comment by amykglen** [comment]
ok, it looks like I successfully just fixed this issue - here's the same query re-run just now: https://arax.ncats.io/?r=40860

what happened is that our production/beta KG2s were using ITRB's production Plover instance (since the ITRB CI instance was down for maintenance the other day). but strangely, the ITRB production Plover had been reverted to KG2.7.4c, even though KG2.7.5c was successfully deployed to ITRB production Plover 12 days ago (I verified after the deployment that KG2.7.5c had been loaded). not sure why this happened, but I just reported it to Kanna in [this thread](https://ncatstranslator.slack.com/archives/C023J4P0Y1M/p1650689445907049).
